### PR TITLE
[bootstrap] Allow unqualified subdirs

### PR DESCRIPTION
**Pull request by rgrinberg**
The bootstrap script changes to allow for subdirs in libraries that allow this.

### CODE REVIEW

File: bootstrap.ml [hunk 1]
```diff
@@ -233,13 +237,32 @@ let cleanup ~keep_ml_file =
...
+
+let find =
+  let rec loop acc = function
+    | [] -> List.sort ~cmp:String.compare acc
+    | p :: ps ->
+      let (files, dirs) =
+        readdir p
+        |> List.partition ~f:Sys.is_directory
+      in
+      loop (ps @ dirs) (List.rev_append files acc)
```
Comment (nojb) [inline bootstrap.ml]: Shouldn't this be `loop (List.rev_append files acc) (ps @ dirs)` ?

Comment (rgrinberg) [reply bootstrap.ml]: Fixed. Thanks.

**Comment by rgrinberg** [comment]
> LGTM. Shouldn't we add the corresponding (include_subdirs) stanza to src/dune/dune ?

It already made its way there in the previous PR :)

**Comment by nojb** [comment]
> It already made its way there in the previous PR :)

Ah, had missed it. Thanks.

**Pull request merged**